api/README.md

```diff
@@ -1 +1,6 @@
-# Express/Node Project
\ No newline at end of file
+# Express/Node Project
+## AUTHENTICATION:
+- /login -> should be working
+- /signup -> Firebase authentication with email and password
+- /users POST -> store user info (name, surname, bio...) on MongoDB
```

Comment:
Well done! Have you created the route for users?

Reply:
Yep! It's in the api/api-routes.js file.